srcpkgs/python3-poetry-core/template

```diff
@@ -1,23 +1,22 @@
 # Template file for 'python3-poetry-core'
 pkgname=python3-poetry-core
-version=1.3.1
-revision=2
+version=1.5.0
+revision=1
 build_style="python3-pep517"
-make_check_args="--deselect tests/masonry/builders/test_sdist.py::test_default_with_excluded_data
- --deselect tests/masonry/builders/test_wheel.py::test_default_src_with_excluded_data"
 depends="python3"
-checkdepends="python3-devel python3-virtualenv python3-pyrsistent python3-pytest-mock git"
+checkdepends="python3-devel python3-setuptools python3-virtualenv python3-pyrsistent
+ python3-pytest-mock"
 short_desc="Poetry PEP 517 Build Backend & Core Utilities"
 maintainer="Kye Shi <shi.kye@gmail.com>"
 license="MIT"
 homepage="https://github.com/python-poetry/poetry-core"
 changelog="https://raw.githubusercontent.com/python-poetry/poetry-core/main/CHANGELOG.md"
 distfiles="https://github.com/python-poetry/poetry-core/archive/refs/tags/${version}.tar.gz"
-checksum=8da8ce8f2e861527f660d7205a77095279d462b44d1d11bcf9e8ce74d96219ee
-make_check_pre="env PYTHONPATH=src"
+checksum=c186b6212224fddae1de8ad7c5e660f40f295cf42559550523f15e72c562f5a8
+make_check_pre="env PYTHONPATH=src PATH=/usr/libexec/chroot-git:${PATH}"
 
 pre_check() {
-	rm -r tests/integration
+	rm -rf tests/integration
```

Comment:
do we need to -f? I prefer not doing it so the line will error when it can be removed

Reply:
I was running the tests multiple times with `./xbps-src check <pkg>` and it was erring in alternate runs. So I had to do `./xbps-src clean <pkg>` every time before running tests. Does that make sense?

Reply:
yes, that is expected behaviour. running steps multiple times without clean isn't really intended